pytanque/protocol.atd

```diff
@@ -23,23 +23,27 @@ type request = {
     params: abstract
 }
 
-type init_params = {
-    ~debug <python default="False">: bool;
-    root: string;
-}
-
 type start_params = {
-    env: int;
```

Comment:
here we also have `?pre_commands: string`, to inject stuff previous to starting the proof.

Reply:
Actually these kind of changes are backwards compatible, so no rush; that's an advantage of records, we can use optional parameters to add new stuff without breaking clients too much.